### ISSUE TITLE
why is `SemFiniteDiff` not part of `AutoDiffSEM`?

### ISSUE THREAD

**Issue by aaronpeikert** [StructuralEquationModels/StructuralEquationModels.jl#111]

**Comment by Maximilian-Stefan-Ernst** [comment]
Because finite difference approximation is not automatic differentiation, and I think it is a good idea to have some functionality in the core package for people that don't want/can specify gradients for their loss function but also dont want to get into automatic differentiation (i.e. want to use the old-school approach)

**Comment by aaronpeikert** [comment]
Makes sense! Let us keep the issue open as a reminder to note `FiniteDiff` as an alternative in the documentation of `AutoDiffSem`.
